dashboard/config/libraries/PoemBot.interpreted.js

```diff
@@ -0,0 +1,68 @@
+function poemBotDraw() {
+  if (isVisible) {
+    drawPoem();
+  }
+}
+
+function drawPoem() {
+  var outerMargin = 50;
+  var defaultLineHeight = 50;
+  var defaultFontSize = 25;
+  var cursor = outerMargin;
+  fill('black');
+  noStroke();
+  textSize(defaultFontSize);
+  textAlign(CENTER);
+  if (poem.title) {
+    textSize(defaultFontSize * 2);
+    text(poem.title, 200, cursor);
+    textSize(defaultFontSize);
+    cursor += defaultLineHeight;
+  }
+  if (poem.author) {
+    cursor -= defaultLineHeight / 2;
+    textSize(16);
+    text("by: " + poem.author, 200, cursor);
+    textSize(defaultFontSize);
+    cursor += defaultLineHeight;
+  }
+
+  var lineHeight = (400 - cursor) / poem.lines.length;
+  for (var i = 0; i < poem.lines.length; i++) {
+    var line = poem.lines[i];
+    var width = textWidth(line);
+    textSize(Math.min(defaultFontSize, defaultFontSize * (400 - outerMargin) / width));
+    text(line, 200, cursor);
+    textSize(defaultFontSize);
+    cursor += lineHeight;
+  }
+}
+
+var poem = {
+  title: undefined,
+  author: undefined,
+  lines: []
+};
+var isVisible = false;
+
+function randomWord() {
+  return "cat";
```

Comment:
TODO: this will return a random word from a list curated by Amy BW but the list is not ready yet, so this is just a placeholder

Reply:
random thought: it could be fun to have it return a random word from a small list?  This would also help make sure that nothing weird happens when it returns different values.

Reply:
To elaborate, I remember long ago seeing something similar where things seemed fine, but actually the random generator was being called quite often, not just at init time, and once a variety of random values were being returned, a piece of UI that was supposed to be stable was actually constantly changing!